primitives/ext/src/lib.rs

```diff
@@ -40,19 +40,22 @@ use moonbeam_rpc_primitives_debug::{
 pub trait MoonbeamExt {
 	// Old format to be deprecated.
 	fn raw_step(&mut self, data: Vec<u8>) {
-		let data: RawStepLog = Decode::decode(&mut &data[..]).unwrap();
-		EventV1::RawStep(data).emit();
+		if let Ok(data) = RawStepLog::decode(&mut &data[..]) {
+			EventV1::RawStep(data).emit();
+		}
```

Comment:
I think we should log when failed to decode so we can debug in case we need.

Reply:
Pushed !